### PR TITLE
create location page mobile version, t2h

### PR DESCRIPTION
**Pull request by salwasiksik**
relates #21

### CODE REVIEW

**Review by ahmedhmmad — DISMISSED** [review]

**Review by MohamedSbaih — COMMENTED** [review]
Add this file inside the overview folder

**Review by azzsal — DISMISSED** [review]

File: locations-page/locations.html [hunk 1]
```diff
@@ -12,7 +12,32 @@
...
+    <!-- add the first photo -->
+    <div class="container">
+      <img src="../assets/images/Location-page/Group 26.svg" alt="Berlin">
+      <div class="content"><p>Travel Berlin <br>Like Never Before</p></div>
+  </div>
+  </main>
+  <!-- add the second photo with caption -->
+  <div><p class="title-of-pic">Featured Domestic Tours</p></div>
+  <div class="container">
+  <img class="sec-pic" src="../assets/images/Location-page/Rectangle 4.svg" alt="sanfrancisco">
```
Comment (Yousef-Dall) [inline locations-page/locations.html:34]: push the image

File: locations-page/locations.html [hunk 2]
```diff
@@ -12,7 +12,32 @@
...
+  <div class="sec-content"><p>San Fransisco</p></div>
+  </div>
+  <!-- add Explor button -->
+  <button><p>Explore</p></button>
```
Comment (Yousef-Dall) [inline locations-page/locations.html:38]: add the link to the destination page

File: locations-page/locations.html [hunk 3]
```diff
@@ -12,7 +12,32 @@
...
+  <!-- add third photo -->
+  <img class="thrid-pic" src="../assets/images/Location-page/Rectangle 6.svg" alt="paris" >
```
Comment (Yousef-Dall) [inline locations-page/locations.html:40]: push the image

File: locations-page/locations.html [hunk 4]
```diff
@@ -12,7 +12,32 @@
...
+      <h1>
+        Here is your<br/>
+        <span class="indented">Holiday Dates</span>
+      </h1>
+    </div>
+  </header>
+  <main>
+    <!-- add the first photo -->
+    <div class="container">
+      <img src="../assets/images/Location-page/Group 26.svg" alt="Berlin">
```
Comment (Yousef-Dall) [inline locations-page/locations.html:27]: push the image

**Review by Yousef-Dall — CHANGES_REQUESTED** [review]

**Review by ahmedhmmad — APPROVED** [review]

**Review by azzsal — APPROVED** [review]

**Review by Yousef-Dall — APPROVED** [review]

**Review by MohamedSbaih — APPROVED** [review]

**Pull request merged**